### PR TITLE
Allow `:` as class prefix-delimeter

### PR DESCRIPTION
**Pull request by tkalmar**
## Description

Allow colon as class prefix. This is useful when working with tailwind and various breakpoints.

## Checklist

- [x] I’ve added tests to confirm my change works.
- [x] (If changing the API or CLI) I’ve documented the changes I’ve made (in the `docs/` directory).
- [x] (If the change is user-facing) I’ve added my changes to `changelog_unreleased/*/XXXX.md` file following `changelog_unreleased/TEMPLATE.md`.
- [x] I’ve read the [contributing guidelines](https://github.com/prettier/prettier/blob/main/CONTRIBUTING.md).

**✨[Try the playground for this PR](https://prettier.io/playground-redirect)✨**

### CODE REVIEW

**Comment by alexander-akait** [comment]
Good catch, can you improve changelog?

File: src/language-html/syntax-attribute.js [hunk 1]
```diff
@@ -60,7 +60,7 @@ function printImgSrcset(value) {
   );
 }
 
-const prefixDelimiters = ["__", "--", "_", "-"];
+const prefixDelimiters = ["__", "--", "_", "-", ":"];
```
Comment (thorn0) [inline src/language-html/syntax-attribute.js:63]: Tailwind classes often contain `-`, which will have higher priority than `:` if implemented this way. Is this intended?

**Prettier pr-10906**
[Playground link](https://deploy-preview-10906--prettier.netlify.app/playground/#N4Igxg9gdgLgprEAuEAeAJgSwG4AIwA2AhgM4kC8AOiALbpIBmEEAtAIy51IBGRATu1wAPAo2bDRABz5wGmIQCZcLFtNnyFKkhBpwYAC0xQA5izgEScXAFcoM4vHQB9QqRJOYEJ3IIEnADgAGF31+EmoAPlQAeixsCJAAGhAISRhMaBJkUH4+CAB3AAV+BCyUIgJ8ogBPLOTuPiIwAGs9AGVJJqNjZBg+azhkuBpuOHR0MYAZIhNrImM4ADEIPhoiGHSTZBAia08kkH0YGgIAdUN4Ek6wODbSzHTsB+rtsDIDo0s+GELG4zXkAwKpZkgArEhCABCjRa7SIukmRjggOBgxA4KEbW6BDgAEVrBB4CiLGjOnwvtsjicDtIjDBTph0AZkEFktIIJZTo1JNs1F9sMjkgBHAnwX6pMo7EgsKBwMZjA4yEWYGS-eYAxio5KWGiYXr9NEkbF40XIzUk5IwIjcBlM-TIBSWxqYAjdADCOg1IDgJAArAdrJYACrWspAi0gbADACSUAmsDaYD4mDSAEE420YNUccTLABfPNAA)

```sh
--parser html
```

**Input:**

```html
<div class="md:foo-1 md:bar-1 xl:foo xl:prefix2 --prefix2--something-else unrelated_class_to_fill_80_chars"></div>
```

**Output:**

```html
<div
  class="
    md:foo-1
    md:bar-1
    xl:foo xl:prefix2
    --prefix2--something-else
    unrelated_class_to_fill_80_chars
  "
></div>

```

Comment (tkalmar) [reply src/language-html/syntax-attribute.js]: You are right. I wasn't aware of that. I changed the order of delimeters and adjusted the test case.
Thanks!

**Review by fisker — APPROVED** [review]

**Comment by fisker** [comment]
thanks for contributing

**Pull request merged**